source/precalculus/source/06-TR/04.ptx

```diff
@@ -12,6 +12,632 @@
+          </p>
+        </introduction>
+
+        <task>
+          <statement>
+            <p>
+              In triangle <m>ABC</m>, <m>A=90</m>,<degree/>, <m>c=3</m>, and <m>B=45</m><degree/>. What is the length of <m>a</m>?
+              <ol marker= "A." cols="2">
+                <li> <m>6</m> </li>
+                <li> <m>5</m></li>
+                <li> <m>4</m> </li>
+                <li> <m>3\sqrt{2}</m> </li></ol>
+              </p>
+          </statement>
+          <answer>
+            <p>
+              D
+            </p>
+          </answer>
+        </task>
+
+        <task>
+          <statement>
+            <p>
+              In triangle <m>ABC</m>, <m>B=90</m>,<degree/>, <m>b=6\sqrt{2}</m>, and <m>C=45</m><degree/>. What is the length of <m>c</m>?
+              <ol marker= "A." cols="2">
+                <li> <m>6</m> </li>
+                <li> <m>5</m></li>
+                <li> <m>4</m> </li>
+                <li> <m>12</m> </li></ol>
+            </p>
+          </statement>
+          <answer>
+            <p>
+              A
+            </p>
+          </answer>
+        </task>
+
+        <task>
+          <statement>
+            <p>
+              In triangle <m>ABC</m>, <m>B=90</m>,<degree/>, <m>b=4</m>, and <m>A=45</m><degree/>. What is the length of <m>c</m>?
+              <ol marker= "A." cols="2">
+                <li> <m>\sqrt{6}</m> </li>
+                <li> <m>2\sqrt{2}</m></li>
+                <li> <m>3</m> </li>
+                <li> <m>6</m> </li></ol>
+            </p>
+          </statement>
+          <answer>
+            <p>
+              B
+            </p>
+          </answer>
+        </task>
+      </activity>
+
+      <activity xml:id="exploring-30-60-90-triangle">
+        <introduction>
+          <p>
+            Suppose you are given a <m>30-60-90</m> triangle, where you know the angle opposite the <m>30</m><degree/> angle is of length <m>1</m>.
+          </p>
+        </introduction>
+        <task>
+          <statement>
+            <p>
+              Draw a right triangle and label the angles to have <m>30</m><degree/>, <m>60</m> <degree/>, and <m>90</m><degree/> and the side opposite of the <m>30</m><degree/> angle as having a length of <m>1</m>.
+            </p>
+          </statement>
+          <answer>
+            <p>
+              Students should be able to draw a right triangle with one <m>90</m><degree/> angle (indicated by a box), one <m>30</m><degree/> angle and one <m>60</m><degree/> angle. The side opposite the <m>30</m><degree/> angle should have a side length of <m>1</m>.
+            </p>
+          </answer>
+        </task>
+        <task>
+          <statement>
+            <p>
+              Unlike the <m>45-45-90</m> triangle, we cannot easily find another side length with the information given. Using trigonometry, which trigonometric function would help us find the length of the hypotenuse with the information given?
```

Comment:
I don't know why, but that just didn't pop into my brain :-p

Reply:
Would you prefer we go with the equilateral triangle route or with the activity (except for that statement)? Or add in another activity?

Reply:
Having them plug into the calculator feels equivalent to me to just telling them what it is.  So I don't love that.  I'd much prefer they do it for themselves since they can (and fairly quickly too in this case).